lib/reek/ast/sexp_extensions.rb

```diff
@@ -409,12 +403,49 @@ def text_name
         end
       end
 
+      # Utility methods for :module nodes.
+      module ModuleNode
+        include ModuleNodeBase
+        def name() self[1] end
+
+        def simple_name
+          name.is_a?(::Parser::AST::Node) ? name.simple_name : name
```

Comment:
I know it's legacy code but can we please document somewhere (here?) with a speaking example what the difference between "name" and "simple_name" is? And then get rid of it asap.
A "name" that has an attribute called "simple_name" is just wrong on so many levels.:)